### ISSUE TITLE
How to train the retina mask  on customer data set

### ISSUE THREAD

**Issue by adeebakausar** [chengyangfu/retinamask#19]

**Comment by monocongo** [comment]
I am also interested to see if there is an example on how to do this. I have a segmentation dataset with masks for a single class that I'd like to use as training data for this model. Please advise, thanks!

**Comment by chengyangfu** [comment]
Hi @monocongo , 
    I follow the COCO setting. If you can format your dataset in COCO format, you should be able to train it. But I haven't updated the code fo r a while. I am worried the new PyTorch code may break the code.

**Comment by Eldad27** [comment]
@chengyangfu  How about the number of classes please? Inside the defaults.py folder, should  '_C.MODEL.ROI_BOX_HEAD.NUM_CLASSES =81' be changed to reflect the exact number of classes for the custom dataset or should it be changed to 'Number of classes + 1'? where 1 = background class? In general, how do I handle number of classes when training on my own dataset?
